made-with.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default.html
+title: Made With Servo
+---
+
+<style type="text/css">
+
+.made-with-container {
+  gap: 20px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  margin-bottom: 32px;
+}
+
+.made-with-item {
+  display: block;
+  width: 400px;
+  flex: 0 0 400px;
+  border: 2px solid #333;
+  cursor: pointer;
+  color: inherit;
+  text-decoration: inherit;
+}
+
+.made-with-item:hover {
+  background-color: rgba(27, 131, 129, 0.5);
```

Comment:
This line has no effect because of the same property below. I don't have a preference for the hover color here. Either way, a CSS variable needs to be created, but I can do that later.